scipy/stats/_stats.pyx

```diff
@@ -513,6 +513,113 @@ cdef double _geninvgauss_pdf(double x, void *user_data) nogil except *:
     return math.exp(_geninvgauss_logpdf_kernel(x, p, b))
 
 
+cdef double _phi(double z) nogil:
+    """evaluates the normal PDF. Used in `studentized range`"""
+    cdef double inv_sqrt_2pi = 0.3989422804014327
+    return inv_sqrt_2pi * math.exp(-0.5 * z * z)
+
+
+cdef double _logphi(double z) nogil:
+    """evaluates the log of the normal PDF. Used in `studentized range`"""
+    cdef double log_inv_sqrt_2pi = -0.9189385332046727
+    return log_inv_sqrt_2pi - 0.5 * z * z
+
+
+cdef double _Phi(double z) nogil:
+    """evaluates the normal CDF. Used in `studentized range`"""
+    # use a custom evaluation because using cs.ndtr results in incorrect PDF
```

Comment:
```suggestion
    # use a custom function because using cs.ndtr results in incorrect PDF
```
Please look for an existing issue and file one if needed. Might be an easy fix for someone who knows what they're doing

Reply:
Found the issue GH-8344, which seems somewhat related. I don't want to hijack it with what is (probably) a different issue, so I'm going to do a bit of hunting myself and see if I can make a different one.

Reply:
Finally got python working in a 32bit linux distro, and there's defiantly a difference compared to 64bit. 

32 bit abs. error with our `_Phi` function:
![err_32bit](https://user-images.githubusercontent.com/45153623/117709754-25a16400-b186-11eb-8318-74ce56b40943.png)

64 bit abs. error with our `_Phi` function:
![err_64bit](https://user-images.githubusercontent.com/45153623/117709759-2803be00-b186-11eb-8292-0c9889426098.png)

Reply:
^Code for generating above 
```cython
cimport cython
cimport scipy.special.cython_special as cs
from libc cimport math

cdef double _Phi(double z) nogil:
    """evaluates the normal CDF. Used in `studentized range`"""
    # use a custom evaluation because using cs.ndtr results in incorrect PDF
    # evaluation at q=0 on 32bit systems
    cdef double m_sqrt1_2 = 0.7071067811865475
    return 0.5 * math.erfc(-z * m_sqrt1_2)
	
	
cpdef double diff(double x):
    return cs.ndtr(x) - _Phi(x)
```

```python
from ndtr_comp import diff
import numpy as np
import matplotlib.pyplot as plt

x = np.linspace(-1, 1, 1000)
y = np.vectorize(diff)(x)

plt.plot(x,y)
plt.show()
plt.savefig("err.png")
```

Reply:
Ok, please also consider the suggested change from "evaluation" to the more specific "function"